ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3058,3 +3058,86 @@ def test_tensorflow_Elu(
         features=x[0],
         name=name,
     )
+
+
+@st.composite
+def _conv2d_helper(draw):
+    input_dtype, \
+        x, \
+        filters, \
+        strides, \
+        dilations, \
+        data_format, \
+        padding = _x_and_filters(draw)
+
+    if padding == "EXPLICIT":
+        explicit_paddings = padding
+        pad_top = draw(st.integers(min_value=0, max_value=3))
+        pad_bottom = draw(st.integers(min_value=0, max_value=3))
+        pad_left = draw(st.integers(min_value=0, max_value=3))
+        pad_right = draw(st.integers(min_value=0, max_value=3))
+
+        if data_format == "NHWC":
+            explicit_paddings = [(0, 0),
+                                 (pad_top, pad_bottom),
+                                 (pad_left, pad_right),
+                                 (0, 0)]
+        else:
+            explicit_paddings = [(0, 0),
+                                 (0, 0),
+                                 (pad_top, pad_bottom),
+                                 (pad_left, pad_right)]
+
+    else:
+        explicit_paddings = []
+
+    return input_dtype, x, filters, dilations, data_format, strides, padding, explicit_paddings
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Conv2D",
+    x_f_d_df=_conv2d_helper(),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_Conv2D(
+    *,
+    x_f_d_df,
+    test_flags,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    input_dtype, \
+        x, \
+        filters, \
+        dilation, \
+        data_format, \
+        stride, \
+        padding, \
+        explicit_paddings = x_f_d_df
+
+    # Broadcast strides and dilations to correct dims for the ground truth
+    # backend func to run correctly
+    stride = _convolution_broadcast_helper(
+        stride, num_spatial_dims=2, channel_index=3, name="strides"
+    )
+    dilation = _convolution_broadcast_helper(
+        dilation, num_spatial_dims=2, channel_index=3, name="dilations"
+    )
+    if padding == "EXPLICIT":
```

Comment:
You don't need to handle `explicit_paddings` here. That will be done inside the actual functions.

Reply:
ok